File: 프로그래머스/1/42748. K번째수/K번째수.kt
class Solution {
    fun solution(array: IntArray, commands: Array<IntArray>): IntArray {
        var answer = intArrayOf()
        for(command in commands){
            var(start,end,k) = command
            var result = array.sliceArray(start-1 until end)
            result.sort()
            println(result.joinToString())
            answer+= result[k-1]

        }
        return answer
    }
}

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    for(const command of commands){
        const [i,j,k] = command
        const sliced =array.slice(i-1,j)
        
        sliced.sort((a,b) => a-b)
        
        console.log(sliced)
        
        answer.push(sliced[k-1])
    }
    return answer;
}